roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -72,6 +72,16 @@ def get_var(task_vars, *keys, **kwargs):
         raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
     return value
 
+def normalized_release(task_vars):
+    """Helper function to normalize the release format to dotted numbers"""
+    release = str(get_var(task_vars, "openshift_release"))
+    if release.startswith('v'):
+        release = release[1:]  # v3.3 => 3.3
+    return release
+
+def normalized_minor_release(task_vars):
```

Comment:
`normalized_minor_release` and `normalized_release` could take just a release string, not `task_vars`.
It becomes easier to compose and test when functions/objects know as much as they need to know, not more.